### DISCUSSION TITLE
Is current home page friendly enough for farmer audience?

### DISCUSSION THREAD

**Discussion by 1993023565** [lsc-hubs/kenya-hub#28] [General]
Home page should have the specific  county area for ease of use by farmers. mapviewer  change oranic to organic,  change PHHH2O To soil  pH

**Comment by pvgenuchten** [comment]
hi @1993023565, thank you for your comment.

This is an interesting discussion you started here, we have different user groups and each user group will benefit from a targeted user interface.

I agree that for famers, many of the content may be overwhelming and a dedicated user interface for specific problems would be much more favourable. Kalro aims to provide such a solution with a range of dedicated apps, which target only a single problem. It would be nice if we can guide relevant users to these apps quickly.

**Comment by 1993023565** [comment]
That is a good thought. We will always  give guide.
